app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        raise ValueError("Command can take only 2 arguments")
```

Comment:
The error message 'Command can take only 2 arguments' is not very informative. It would be better to specify what command you are talking about. For example: 'cp command can take only 2 arguments'.